### PR TITLE
BAN-66: Add open-sans font to storybook

### PR DESCRIPTION
**Pull request by chavlji**
[jira](https://reciprocitylabs.atlassian.net/browse/BAN-66)

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    use: [
+      {
+        loader: 'file-loader',
+        query: {
+          name: '[name].[ext]'
+        }
+      }
+    ],
+    include: path.resolve(__dirname, '../')
+  }); */
```
Comment (f-jauregui) [inline .storybook/webpack.config.js:25]: What are these comments for?

File: package.json [hunk 2]
```diff
@@ -63,8 +68,9 @@
   },
   "license": "MIT",
   "dependencies": {
-    "lodash": "~4.17.20",
-    "@stencil/core": "~2.3.0"
+    "@stencil/core": "~2.3.0",
+    "copy-webpack-plugin": "^6.3.2",
```
Comment (f-jauregui) [inline package.json:72]: Move it to devDependencies

**Review by f-jauregui — CHANGES_REQUESTED** [review]
Not super happy about compiling CSS again, but it's ok to keep track of the css bundle size and we can remove it later.
Then just these little comments below.

File: package.json (hunk 2 above)
Comment (f-jauregui) [reply package.json:72]: > "copy-webpack-plugin": "^6.3.2",

File: .storybook/webpack.config.js (hunk 1 above)
Comment (chavlji) [reply .storybook/webpack.config.js:25]: 😅

**Pull request merged**